ambry-cloud/src/test/java/com.github.ambry.cloud/CloudBlobStoreTest.java

```diff
@@ -199,28 +211,38 @@ public void testExceptionalDest() throws Exception {
    * @param expiresAtMs the expiration time.
    * @param accountId the account Id.
    * @param containerId the container Id.
+   * @param encrypted the encrypted bit.
    * @return the generated {@link BlobId}.
    * @throws StoreException
    */
   private BlobId addBlobToSet(MockMessageWriteSet messageWriteSet, long size, long expiresAtMs, short accountId,
-      short containerId) {
-    BlobId id = getUniqueId(accountId, containerId);
+      short containerId, boolean encrypted) {
```

Comment:
Because we'll be using the message header to make encrypted decisions, that may need to be reflected here